Formula/haproxy.rb

```diff
@@ -32,6 +32,40 @@ def install
     bin.install "haproxy"
   end
 
+  def caveats; <<~EOS
```

Comment:
these caveats aren't needed

Reply:
I'd like to at least let the user know that it won't work without a config file at `/usr/local/etc/haproxy.cfg`.

Is there some easy way I can include my own sample config in the package?

Reply:
Actually, with the `-f path/to/config` added to `plist_options` it's quite clear to the user which config file it's using, so I agree, I can remove these caveats.